src/nfd_win.cpp

```diff
@@ -250,13 +250,20 @@ nfdresult_t SetDefaultPath(IFileDialog* dialog, const nfdnchar_t* defaultPath) {
 
     Release_Guard<IShellItem> folderGuard(folder);
 
-    // SetDefaultFolder() might use another recently used folder if available, so the user doesn't
-    // need to keep navigating back to the default folder (recommended by Windows). change to
-    // SetFolder() if you always want to use the default folder
+#ifdef nfd_OVERRIDE_RECENT_WITH_DEFAULT
```

Comment:
```suggestion
#ifdef NFD_OVERRIDE_RECENT_WITH_DEFAULT
```

This probably needs to be changed too?

Reply:
Whoops, have that locally, meant to commit it 😅